File: src/main/java/no/nav/syfo/util/Toggle.java
package no.nav.syfo.util;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

@Component
public class Toggle {

    public static boolean enableNullstill;

    public Toggle(
            @Value("${toggle.enable.nullstill:false}") boolean enableNullstill
    ) {
        Toggle.enableNullstill = enableNullstill;
    }
}

File: src/main/kotlin/no/nav/syfo/toggle/Toggle.kt
package no.nav.syfo.toggle

import org.springframework.beans.factory.annotation.Value
import org.springframework.stereotype.Component

@Component
class Toggle(
    @Value("\${toggle.enable.nullstill:false}") enableNullstill: Boolean
) {
    companion object {
        var enableNullstill = false
    }

    init {
        Companion.enableNullstill = enableNullstill
    }
}
